web/server/api/endpoints/lineage.py

```diff
@@ -27,16 +28,25 @@ def _get_table(node: Node, dialect: t.Optional[DialectType] = None) -> t.Optiona
     table: t.Union[exp.Table, str] = node.alias or node.name
     if isinstance(node.expression, exp.Table):
         table = node.expression
+    elif isinstance(node.expression, exp.Alias):
+        table = node.expression.this.table
 
     try:
-        return exp.table_name(table, identify=True, dialect=dialect)
+        return normalize_model_name(table, None, dialect=dialect)
```

Comment:
why are you passing None?

Reply:
Sorry, the PR wasn't done yet. I'm passing in default catalog now.